docs/platforms/android/features/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Features
+sidebar_order: 0
+description: "Learn more about Sentry's Android SDK features."
+---
+
+<Note>
+
+Sentry's Android SDK enables automatic reporting of errors and exceptions, and identifies performance issues in your application.
+
+</Note>
```

Comment:
```suggestion

Sentry's Android SDK enables automatic reporting of errors and exceptions, and identifies performance issues in your application. The below, is a list of features that are available as part of this SDK.

```
I think this works fine as regular text as opposed to a note :)
Question - I believe some of these features are out-of-the-box, while others have to be specially configured? If so, might be nice to divide them so it's clear to the users which are which.

Reply:
> I believe some of these features are out-of-the-box, while others have to be specially configured? If so, might be nice to divide them so it's clear to the users which are which.

Thanks, that's a good suggestion - I'll follow up on that across the Mobile SDKs in separate issues/PRs